File: TripPartner.WebAPI/Domain_Models/Story.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;
using TripPartner.WebAPI.Models;

namespace TripPartner.WebAPI.Domain_Models
{
    public class Story
    {
        [Key]
        public int Id { get; set; }
        [DisplayName("Story made")]
        public DateTime DateMade { get; set; }

        [DisplayName("Last edited")]
        public DateTime LastEdit { get; set; }
        [DisplayName("Story date")]
        public DateTime Date { get; set; }
        [DisplayName("Text")]
        public string Text { get; set; }
        public string CreatorId { get; set; }
        [ForeignKey("CreatorId")]
        public ApplicationUser Creator { get; set; }
        public int TripId { get; set; }
        [ForeignKey("TripId")]
        public Trip Trip { get; set; }

    }
}

File: TripPartner.Web/app/components/stories/services/Story.js
﻿(function (module) {

    module.factory('Story', function () {
        function Story(model) {
            this.Id = model.Id,
            this.DateMade = model.DateMade,
            this.LastEdit = model.LastEdit,
            this.Date = model.Date,
            this.Text = model.Text,
            this.CreatorId = model.CreatorId,
            this.CreatorUsername = model.CreatorUsername,
            this.TripId = model.TripId,
            this.Title = model.Title,
            this.Rating = model.Rating,
            this.Rates = model.Rates
        }
        return Story;
    });

})(angular.module('app.stories'));